cpp/jvm-enclave-common/stubs/stdio.cpp

```diff
@@ -154,4 +150,9 @@ extern "C" {
         }
         return res;   
     }    
+
+    char *fgets(char *s, int size, FILE *stream) {
+        enclave_trace("fgets\n");
+        return NULL;
```

Comment:
nullptr

Reply:
This is weird somehow, this is C code that gets compiled with C++ compiler. `man fgets` obviously does not mention `nullptr` but only `NULL`. For consistency with the remaining code we have, I would agree to use `nullptr`

Reply:
Ah, thought it might be something weird like this, just checking!